src/oauth.ts

```diff
@@ -25,21 +25,21 @@ export function initOAuth() {
 		Config().twitter.consumerKey,
 		Config().twitter.consumerSecret,
 		"1.0A",
-		"oob",
-		"HMAC-SHA1"
+		null,
+		"HMAC-SHA1",
 	);
 }
 
-export async function getOAuthUrl(): Promise<IOAuthData> {
+export async function getOAuthUrl(callback: string = "oob"): Promise<IOAuthData> {
 	return new Promise((resolve, reject) => {
 		oauth.getOAuthRequestToken(
-			{"x_auth_access_type": "dm"},
-			(error, oAuthToken, oAuthTokenSecret) => {
+			{x_auth_access_type: "dm", oauth_callback: callback},
```

Comment:
can you verify that pin auth still works with the bot provisioner using this method?

Reply:
Yeah works fine, it defaults to `"oob"` like before

Reply:
Great, thanks!